packages/notifier/src/publish-kit.js

```diff
@@ -171,6 +118,10 @@ export const makePublishKit = () => {
     }
   };
 
+  const makeUpdateRecord = weakMemoizeUnary(
+    makeUpdateRecordFromPublicationRecord,
+  );
```

Comment:
Do we have a convention for how to name inner vs. outer functions in this kind of wrapping? It seems like they both want to be "makeUpdateRecordFromPublicationRecord", or at least that the wrapped one mentions [heap-based] memoization.

Reply:
I don't know of an explicit convention.  Personally, I use shorter names for inner identifiers.  In any case, I mentioned the memoization.